tslint.json

```diff
@@ -3,6 +3,7 @@
 		"build/lib/tslint"
 	],
 	"rules": {
+		"no-string-throw": true,
```

Comment:
The rule will catch all kinds of string literals and concatenation of strings. It will probably not trigger if you save the string to a variable first. It sure could be extended to do that, but I am not familiar enough with custom TSLint rules to do that

https://sourcegraph.com/github.com/palantir/tslint@master/-/blob/src/rules/noStringThrowRule.ts#L61:11